doc/en/fixture.rst

```diff
@@ -301,6 +301,27 @@ are finalized when the last test of a *package* finishes.
     Use this new feature sparingly and please make sure to report any issues you find.
 
 
+Dynamic scope
+^^^^^^^^^^^^^
+
+In some cases, you might want to change the scope of the fixture without changing the code.
+To do that, pass a callable to ``scope``. The callable must return a string with a valid scope
+and will be executed only once during the fixture definition. It will be called with two
+keyword arguments - ``fixture_name`` as a string and ``config`` with a configuration object.
+
+See the example below:
+
+.. code-block:: python
+
+    def dynamic_scope(fixture_name, config):
```

Comment:
We could use an example of a fixture using this function... but I'm struggling to come up with a good example for this.

I've asked for help in https://github.com/pytest-docker-compose/pytest-docker-compose/issues/3#issuecomment-524283826, let's see if somebody steps up! 👍

Reply:
OK, I've changed the example to less abstract.